### PR TITLE
[B] Adding Vendor Details Endpoint

### LINKED ISSUE #391: [BACKEND] Implement Vendor Details Endpoint

### PR DESCRIPTION
**Pull request by bcetin**
## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Description

- Vendor Details Endpoint Implemented

# Test results
```
System check identified no issues (0 silenced).
1
.2
.
----------------------------------------------------------------------
Ran 2 tests in 3.793s

OK
Preserving test database for alias 'default'...
```
## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have checked my code and corrected any misspellings
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by bekir96 — COMMENTED** [review]
Can you add serializer for vendor_details response? It makes our code mechanism more robust.

**Review by bekir96 — COMMENTED** [review]
And also can you increase label number to better understand what is done at first sight

**Review by bekir96 — CHANGES_REQUESTED** [review]
Can you resolve conflict

File: ecommerce/backend/API/views/vendor.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import json
+
+from ..utils import permissions, Role
+from ..models import Vendor,ImageUrls
+
+@api_view(['GET'])
+@permission_classes([permissions.AllowAnonymous]) 
+def vendor_details(request,vendor_id):
+    #print(vendor_id)
+    vendor = Vendor.objects.filter(id=vendor_id).first()
```
Comment (omerfdeniz) [inline ecommerce/backend/API/views/vendor.py:13]: please check whether that vendor really exists and return 400_BAD_REQUEST if it does not.
```suggestion
    vendor = Vendor.objects.filter(id=vendor_id).first()
    if vendor is None:
        return Response(status=status.HTTP_400_BAD_REQUEST)
```

Comment (omerfdeniz) [reply ecommerce/backend/API/views/vendor.py:13]: Please don't forget to delete debug comments.

**Review by omerfdeniz — CHANGES_REQUESTED** [review]
Please check the availability of the vendor

**Pull request merged**